pkg/kubernetes/kubernetes.go

```diff
@@ -101,6 +101,16 @@ func (k Kubernetes) GetSecret(secretName, namespace string, ctx context.Context)
 	return secret, err
 }
 
+// GetPVC returns a PVC bound with given PVC name
+func (k Kubernetes) GetPVC(pvcName, namespace string, ctx context.Context) (*corev1.PersistentVolumeClaim, error) {
+	pvc := &corev1.PersistentVolumeClaim{}
+	err := k.Client.Get(ctx, types.NamespacedName{Namespace: namespace, Name: pvcName}, pvc)
+	if err != nil {
```

Comment:
As we don't need this method in the operator itself, only the testsuite, I would move this to `test/e2e/utils/kubernetes.go`.

We can then simplify the method to something like:

```go
func (k TestKubernetes) GetPVC (pvcName, namespace string) *corev1.PersistentVolumeClaim {
   pvc := &corev1.PersistentVolumeClaim{}
   err := k.Client.Get(ctx, types.NamespacedName{Namespace: namespace, Name: pvcName}, pvc)
   ExpectNoError(err)
   return pvc
}
```

Reply:
k, makes sense.